assets/src/scripts/monitoring-location/components/hydrograph/method-picker.js

```diff
@@ -28,37 +34,57 @@ const updateAvailableMethods = function(selectElem, methods, store) {
     }
 
     methods.forEach((method) => {
-        selectElem.append('option')
-            .text(method.methodDescription ? `${method.methodDescription}` : 'None')
+        const hasPointsInTimeSpan = method.pointCount > 0;
+        container.append('option')
+            .text(`${method.methodDescription}`)
+            .attr('class', 'method-option sampling-method-selection')
             .attr('selected', method.methodID === selectedMethodID ? true : null)
-            .node().value = method.methodID;
+            .attr('value', method.methodID)
+            .property('disabled', !hasPointsInTimeSpan ? true : null);
         });
 };
 
-/*
+/**
  * Draw the method picker. It will be set initially to the preferred method id if not already
  * set to a specific, available method id. The picker will be hidden if only one method
  * is available for the IV data.
- * @param {D3 selection} elem
- * @param {Redux store} store
+ * @param {D3 selection} container - An HTML element to serve as the base for attaching other elements.
+ * @param {String} parameterCode - The five digit USGS that is unique for each parameter.
+ * @param {Redux store} store - A complex JavaScript object that maps the current state of the application.
  */
-export const drawMethodPicker = function(elem, store) {
-    const pickerContainer = elem.append('div')
+export const drawMethodPicker = function(container, parameterCode, store) {
+    const gridRowSamplingMethodSelection = container.append('div')
+        .attr('id', 'primary-sampling-method-row')
+        .attr('class', 'grid-container grid-row-inner sampling-method-selection');
```

Comment:
Can we use mixins rather than utility classes for grid-container, grid-row (below). Is grid-row-inner needed (we can't do the styling with standard USWDS classes/mixins)?